### PR TITLE
don't use step_x or step_y

### PR DESCRIPTION
**Pull request by Intigracy**
use pixel_x or pixel_y

@Sprok0

### CODE REVIEW

File: maps/randomvaults/AIsat.dmm [hunk 1]
```diff
@@ -62,7 +62,7 @@
 "bj" = (/obj/item/stack/rods,/turf/simulated/floor/plating/airless/damaged,/area/vault/AIsat)
 "bk" = (/obj/structure/closet/crate{icon_state = "crateopen"; opened = 1},/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"},/turf/simulated/floor/plating/airless,/area/vault/AIsat)
 "bl" = (/obj/structure/cable{d1 = 2; d2 = 8; icon_state = "2-8"},/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"},/turf/simulated/wall/r_wall,/area/vault/AIsat)
-"bm" = (/obj/item/device/radio/intercom{name = "Custom Channel"; step_y = 25; pixel_x = -27; pixel_y = 4; listening = 0; freerange = 1},/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"},/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"; tag = ""},/turf/simulated/wall/r_wall,/area/vault/AIsat)
+"bm" = (/obj/item/device/radio/intercom{name = "Custom Channel"; pixel_y = 25; pixel_x = -27; pixel_y = 4; listening = 0; freerange = 1},/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"},/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"; tag = ""},/turf/simulated/wall/r_wall,/area/vault/AIsat)
```
Comment (Exxion) [inline maps/randomvaults/AIsat.dmm:65]: Gee Bill, how come your mom lets you have TWO `pixel_y`s?

**Comment by Exxion** [comment]
And that's why you wait for Travis to finish before merging even hotfixes

**Comment by clusterfack** [comment]
Travis wouldn't have caught it you silly person

**Comment by PsiOmegaDelta** [comment]
`- (! grep 'step_[xy]' maps/**/*.dmm)` in .travis.yml let Bay fail builds with maps that use step_.

**Comment by clusterfack** [comment]
Yeah I was just saying we should change it to do this on our discord

**Pull request merged**